client/keys.go

```diff
@@ -27,7 +27,7 @@ type Key struct {
 	pubArea tpm2.Public
 	pubKey  crypto.PublicKey
 	name    tpm2.Name
-	session session
+	session Session
```

Comment:
nit: could you make this a getter?

Reply:
Could you elaborate on the problem here? I exported the type / interface of `Session` in order to use it for `LoadCachedKey` as a parameter. The actual field in the struct is still unexported.

Reply:
Not a big deal: it would just be nice if someone wanted to call LoadCachedKey with an unknown Session type. This shouldn't be the case now